toolkit/tools/pkg/simplechroottool/simplechroottool.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// A common interface for creating a basic spec/rpm parsing environment inside a chroot.
+
+package simplechroottool
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/rpm"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/safechroot"
+)
+
+const (
+	chrootSpecDirPath = "/SPECS"
+)
+
+// SimpleChrootTool is a tool for creating an environment inside a chroot suitable for basic tasks like parsing RPMs.
+// The following fields will be set by InitializeChroot():
+//   - chroot: The chroot environment we are working in, call SimpleChrootTool.RunInChroot() to execute commands inside the chroot.
+//   - chrootSpecDir: The relative path to the SPECS directory inside the chroot: "/SPECS."
+//   - runChecks: Whether or not to parse with check sections enabled.
+//   - distTag: The distribution tag to use when parsing RPMs.
+type SimpleChrootTool struct {
+	chroot        *safechroot.Chroot
+	chrootSpecDir string
```

Comment:
Is this needed as a separate member, since we're using the constant? Do we want it to be settable somehow?